modules/TerraformController.py

```diff
@@ -306,13 +301,31 @@ def list_machines(self):
         print()
         print('Status Virtual Machines\n')
         if len(response) > 0:
+            # grab the ips
+            for machine in response:
```

Comment:
For loop need to be changed to iterate over all machines. Here some Pseudo Code:
boxes = [phantom_server, windows_domain_controller, windows_server, kali_machine, windows_client , zeek_sensor, osquery_machine]
for box in boxes:
  if config[box] == 1:
    ip = getIP()
    print table